openfisca_us/variables/finance/tax/outputs.py

```diff
@@ -332,20 +244,96 @@ class sep(Variable):
     )
 
 
-class sey(Variable):
+class filer_sey(Variable):
     value_type = float
     entity = TaxUnit
     definition_period = YEAR
-    documentation = (
-        """search taxcalc/calcfunctions.py for how calculated and used"""
-    )
+    documentation = """sey for the tax unit (excluding dependents)"""
+
+    def formula(tax_unit, period, parameters):
+        return tax_unit.sum(
+            tax_unit.members("sey", period)
+            * tax_unit.members("is_tax_unit_dependent", period)
+        )
+
+
+class basic_standard_deduction(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Basic standard deduction"
+    definition_period = YEAR
+
+    def formula(tax_unit, period, parameters):
+        STD = parameters(period).tax.deductions.standard
+        MARS = tax_unit("MARS", period)
+        MIDR = tax_unit("MIDR", period)
+
+        c15100_if_DSI = max_(
+            STD.dependent.additional_earned_income
+            + tax_unit("filer_earned", period),
+            STD.dependent.amount,
+        )
+        basic_if_DSI = min_(STD.amount[MARS], c15100_if_DSI)
+        basic_if_not_DSI = where(MIDR, 0, STD.amount[MARS])
+        basic_stded = where(
+            tax_unit("DSI", period), basic_if_DSI, basic_if_not_DSI
+        )
+        return basic_stded
+
+
+class aged_blind_extra_standard_deduction(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Aged and blind standard deduction"
+    definition_period = YEAR
+
+    def formula(tax_unit, period, parameters):
+        STD = parameters(period).tax.deductions.standard
+        MARS = tax_unit("MARS", period)
+        MARSType = MARS.possible_values
+        num_extra_stded = (
```

Comment:
I'd break this up for clarity
```suggestion
        blind_head = tax_unit("blind_head", period) * 1
        blind_spouse = tax_unit("blind_spouse", period) * 1
        aged_head = (tax_unit("age_head", period) >= STD.aged_or_blind.age_threshold) * 1
        aged_spouse = ((MARS == MARSType.JOINT) & (tax_unit("age_spouse", period) >= STD.aged_or_blind.age_threshold)) * 1
        num_extra_ded = blind_head + blind_spouse + aged_head + aged_spouse
        return num_extra_stded * STD.aged_or_blind.amount[MARS]
```

Reply:
Done